ivy/func_wrapper.py

```diff
@@ -60,19 +60,21 @@ def try_array_function_override(func, overloaded_args, types, args, kwargs):
     )
 
 
-def _get_first_array(*args, **kwargs):
+def _get_first_array(*args, array_fn=None, **kwargs):
```

Comment:
I'm not sure if we should modify the `func_wrapper` here to achieve this, as is there any use case of that other than for the frontend testing? If not let's have that logic in the testing it self, @VedPatwardhan do you think this adds an overhead?